.remarkignore

```diff
@@ -0,0 +1 @@
+/CODE_OF_CONDUCT.md
```

Comment:
Can't we make it that the file just validates?

Reply:
I cut and pasted it directly from the original, and didn't want to modify it so it does every time.